droplets.go

```diff
@@ -320,6 +321,18 @@ func (s *DropletsServiceOp) List(ctx context.Context, opt *ListOptions) ([]Dropl
 	return s.list(ctx, path)
 }
 
+// ListByName lists all Droplets filtered by name returning only exact matches.
+// It is case-insensitive
+func (s *DropletsServiceOp) ListByName(ctx context.Context, tag string, opt *ListOptions) ([]Droplet, *Response, error) {
```

Comment:
I think `name` is a better argument name instead of `tag`.
```suggestion
func (s *DropletsServiceOp) ListByName(ctx context.Context, name string, opt *ListOptions) ([]Droplet, *Response, error) {
```

Reply:
:facepalm: Thanks for catching. Just a copypasta error.  Fixed.